proto/klyraprotocol/bridge/bridge_event.proto

```
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package klyraprotocol.bridge;
+
+import "gogoproto/gogo.proto";
```

buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)